api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -24,12 +24,26 @@ def add_item
             end
           end
 
+          def remove_item
+            if spree_current_order.nil?
+              render json: { error: "Order doesn't exist" }, status: 404
+            else
+              line_item = spree_current_order.line_items.find(params[:line_item_id])
+
+              spree_authorize! :update, spree_current_order, order_token
+
+              dependencies[:remove_item_from_cart].call(order: spree_current_order, line_item: line_item)
```

Comment:
```
dependencies[:remote_item_from_cart].call(
  order:     spree_current_order,
  line_item: line_item
)
```
It look more readable in my opinion

Reply:
@kushniryb we should always operate on the current order here, so IMHO it shouldn't be configurable

Reply:
Yeah but we still need to pass it to the service object since `spree_current_order` is accessible only in controllers & views, isn't it?
As for the comment - I've meant just formatting.